dist/index.d.ts

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
```

Comment:
Build artifacts should **not** be checked in to source control. This includes the dist directory

Reply:
Whoops! I added  dist to git ignore after this slipped through